code/modules/storage/storage.dm

```diff
@@ -69,6 +69,9 @@
 	if (istype(src.linked_item, /obj/item))
 		var/obj/item/I = src.linked_item
 		I.tooltip_rebuild = TRUE
+		// Items with storage datums attached shouldn't be able to be used for stealthy pickpocketing
+		if (!(I.item_function_flags & OBVIOUS_INTERACTION_BAR))
+			I.item_function_flags += OBVIOUS_INTERACTION_BAR
```

Comment:
while it's safe here because you are checking for the flag already, best practice is still to use bitwise operations when working with bitfields rather than addition/subtraction